m4_sessions/1803-inning/group_1.md

```diff
@@ -1,13 +1,23 @@
 Group 1
 
-Group Members:
+Group Members: James Logue, Melissa Wagasky, Melena Suliteanu
 
-Name of Session:
+Name of Session: Your Smartphone is a Civil Rights Issue: Digital Privacy as a Developer
 
 Session Description (Why should students come to this session):
+If you care about digital privacy and your role in contributing to good practices.
 
 Session Outcomes (What will students learn/leave with?):
+You will walk away with an understanding of the current landscape of digital privacy, the companies in the privacy ecosystem, and a manifesto around your own beliefs of privacy as a developer.
```

Comment:
Excellent!  We have done a few privacy sessions in the past.  Look back in the Gear Up folders or I can point you to them!